Dockerfile

```diff
@@ -14,10 +14,17 @@ EXPOSE 8096
 VOLUME /config /media
 
 ARG FFMPEG_URL=https://www.johnvansickle.com/ffmpeg/old-releases/ffmpeg-4.0.3-64bit-static.tar.xz
-RUN apt update \
- && apt install -y xz-utils \
- && curl ${FFMPEG_URL} | tar Jxf - -C /usr/bin --wildcards --strip-components=1 ffmpeg*/ffmpeg ffmpeg*/ffprobe \
- && apt remove -y xz-utils \
- && apt install -y libfontconfig1  # needed for Skia
+RUN DEBIAN_FRONTEND=noninteractive \
+ apt-get update && \
+ apt-get upgrade -y && \
```

Comment:
Please drop the upgrade. This makes the package too large, since the layers will not be merged down.

Reply:
+1 - Please remove.

Reply:
Fair enough I guess. I've had some very very static containers riddled with known security problems, so I tend to default to using upgrade.